### DISCUSSION TITLE
Is it possible to hide the "paging" info in the history list view?

### DISCUSSION THREAD

**Discussion by ChristopherHaws** [lzybkr/PSReadLine#3679] [Q&A]
Hello,

I was wondering if it is possible to remove this portion of the history prediction results? When using an IDE, vertical terminal space is pretty limited so I would like to hide it.
```shell
<-/10>                     <History(10)>
```
![image](https://github.com/PowerShell/PSReadLine/assets/5934505/3d79e191-1194-4c54-ad4a-3885998b500e)

Thanks!

**Comment by daxian-dbw** [comment]
It cannot be hidden as of today, but I guess we can have a setting to allow hiding the metadata line.

**Reply by RalfReddings** [reply]
I found myself being hindered by this "Paging" thing allot too. Quite often, my terminal window is very narrow and this "paging" line just takes important space.

Looking forward for an option to turn it of too. Cheers

**Comment by johan-lindqvist** [comment]
Would really appreciate it if the metadata line was configurable to be hidden. Seems like a strange thing to force onto users.
